pkg/apiserver/middlewares/v1/jwt.go

```diff
@@ -213,18 +222,23 @@ func (j *JWT) Authenticator(c *gin.Context) (interface{}, error) {
 		}
 	}
 
+	clientIP := c.ClientIP()
+	if clientIP == "" && isUnixSocket(c) {
```

Comment:
But what if the request is proxied to unix socket from a web server? we need to set the remote addr on the request before c.ClientIP() is called.

https://github.com/gotify/server/blob/46281d6a51dc0ca6a2ac13f81db89224b990d876/router/router.go#L34-L39

Reply:
so it's the proxy that sets "@" as remoteaddr? and I have to copypasta the 3-line middleware from gotify, right?

Reply:
Guess it only works with g.ForwardedByClientIP = true